src/test/java/core/basesyntax/service/impl/DataParsedImplTest.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ParserService;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class DataParsedImplTest {
+    private static List<String> data;
+    private static final FruitTransaction FIRST =
+            new FruitTransaction(FruitTransaction.Operation.BALANCE, "banana", 20);
+    private static final FruitTransaction LAST =
+            new FruitTransaction(FruitTransaction.Operation.SUPPLY, "banana", 100);
+    private final ParserService parserService = new DataParsedImpl();
+    private List<FruitTransaction> fruitTransactionList;
+    private List<FruitTransaction> emptyList;
```

Comment:
instantiate object that need specific test inside the test case. 
```suggestion
```

Reply:
What to do then? My DataParser returns List<FruitTransaction>.

Reply:
List of FruitTransaction*

Reply:
All questions should be discussed in Java Chat, write there, please

Reply:
Let's make them local variables, it ensures each test is completely independent of others. There's no shared state, so the tests are less likely to interfere with each other. This is often considered a best practice in unit testing